src/http/client.rs

```diff
@@ -62,6 +62,14 @@ impl HTTPClientDataSource {
 }
 
 impl DeferredDataSource for HTTPClientDataSource {
+    fn fetch_description(&mut self) {
+
+    }
+
+    fn get_description(&mut self) -> DataSourceDescription {
+        DataSourceDescription {source_locator: None}
+    }
```

Comment:
same questions as above

Reply:
`baseurl` is fine here.

Reply:
https://github.com/StanfordLegion/prof-viewer/pull/36/commits/c28219e2ce499d54bda9492294abdbf073034ced